ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -328,3 +328,7 @@ def fmin(self, y, name=None):
     def minimum(self, y, name=None):
         y_ivy = _to_ivy_array(y)
         return ivy.minimum(self._ivy_array, y_ivy)
+
+    @with_supported_dtypes({"2.5.0 and below": ("int32", "int64")}, "paddle")
+    def floor_divide(self, y, name=None):
+        return ivy.floor_divide(self._ivy_array, y)

```

Comment:
This was supposed to be `y._ivy_array`. Maybe a mistake?

Reply:
I have tried to explicitly convert passed argument y to an ivy_array using the y._ivy_array attribute but on testing, I  am getting an error saying the paddle.Tensor object has no attribute ._ivy_array so I have reverted back the changes.
May

Reply:
for more info to see how the tests are failing<img width="1048" alt="Screenshot 2023-06-28 at 13 59 50" src="https://github.com/unifyai/ivy/assets/38653871/c64067f7-9736-4267-a3e4-70ed9795862c">